### DISCUSSION TITLE
Canvas In React Native

### DISCUSSION THREAD

**Discussion by feniljariwala82** [react-native-community/discussions-and-proposals#441] [Ideas]
Is it possible to **draw and clear images in react native** apps? Basically, I'm have Canvas kind of use-case. I have 256 images. Now I want to display all the images one by one with some millisecond gap. We can do this using Canvas in web-apps. So my question is can we do the same in react native apps? Can anyone guide me here?

**Comment by slorber** [comment]
Looks like a good use-case for https://github.com/Shopify/react-native-skia ?
